eventsourced/src/lib.rs

```diff
@@ -352,25 +377,32 @@ where
     Self: Debug,
     E: EventSourced,
 {
-    fn handle_cmd(&self, id: &E::Id, state: &E) -> Result<E::Evt, BoxedAny>;
-
-    fn make_reply(&self, id: &E::Id, state: &E, evt: E::Evt) -> BoxedAny;
+    #[allow(clippy::type_complexity)] // internal type, who cares?
```

Comment:
`Box<dyn FnOnce(&E) -> BoxedAny + Send + Sync>` just occurs repeatedly, hence introducing a well named type alias would be nice.